### PR TITLE
Cellprofiler interactivetool bug fixes and naming

### PR DESCRIPTION
**Pull request by sunyi000**
cellprofiler bug fixes and naming convention change

### CODE REVIEW

**Comment by bgruening** [comment]
There is a merge conflict here.

**Comment by sunyi000** [comment]
> There is a merge conflict here.

is this fixed now? thanks.

File: tools/interactive/interactivetool_cellprofiler.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<tool id="interactive_tool_cellprofiler" tool_type="interactive" name="cellprofiler" version="@VERSION@+galaxy0" profile="23.1">
-	<description>cell image analysis software</description>
+<tool id="interactive_tool_cellprofiler" tool_type="interactive" name="Run CellProfiler" version="@VERSION@+galaxy0" profile="23.1">
+	<description>interactive tool</description>
```
Comment (bgruening) [inline tools/interactive/interactivetool_cellprofiler.xml:2]: interactive tool is a very technical term ... I liked the previous version more. Happy to merge as it is if you prefer this version.

Comment (beatrizserrano) [reply tools/interactive/interactivetool_cellprofiler.xml:2]: We discussed it here with Leonid. To follow the [naming convention agreed at the BH](https://github.com/elixir-europe/biohackathon-projects-2023/blob/main/16/conventions.md) and distinguish it from the the CellProfiler tools, that seems to be the best option. 

Cause it does not look very self-explanatory now:
Screenshot 2024-02-29 at 15 53 40

**Review by bgruening — APPROVED** [review]

**Pull request merged**